src/serve/index.js

```diff
@@ -54,6 +50,8 @@ exports.serveWeb = function (opts, cb) {
   app.use('/compile/', importMiddleware(getPath('static/')));
 
   // serve static files
+  var devkitPath = path.resolve(__dirname, '..', '..');
+  app.use('/devkit/', express.static(devkitPath));
```

Comment:
Is it really? I thought only module code was available, and then only the folders specified in a module's package config.